pyquil/paulis.py

```diff
@@ -140,52 +142,52 @@ def __mul__(self, term):
     def __rmul__(self, other):
         """Multiplies this PauliTerm with another object, probably a number.
 
-        :param Number term: A number to multiply by
+        :param other: A number or PauliTerm to multiply by
         :returns: A new PauliTerm
         :rtype: PauliTerm
         """
         assert isinstance(other, Number)
         return self * other
 
-    def __add__(self, term):
+    def __add__(self, other):
         """Adds this PauliTerm with another one.
 
-        :param PauliTerm term: A PauliTerm object
+        :param other: A PauliTerm object or a Number
         :returns: A PauliSum object representing the sum of this PauliTerm and term
         :rtype: PauliSum
         """
-        if isinstance(term, Number):
-            return self + PauliTerm("I", 0, term)
-        elif isinstance(term, PauliSum):
-            return term + self
+        if isinstance(other, Number):
+            return self + PauliTerm("I", 0, other)
+        elif isinstance(other, PauliSum):
+            return other + self
         else:
-            new_sum = PauliSum([self, term])
+            new_sum = PauliSum([self, other])
             return new_sum.simplify()
 
-    def __radd__(self, term):
+    def __radd__(self, other):
         """Adds this PauliTerm with a Number.
 
-        :param Number term: A number to multiply by
+        :param other: A PauliTerm object or a Number
         :returns: A new PauliTerm
         :rtype: PauliTerm
         """
-        assert isinstance(term, Number)
-        return self + term
+        assert isinstance(other, Number)
```

Comment:
I also realize this wasn't part of this PR, so it's fine if you don't want to change it here- I just think it should be changed at some point.